preprocessy/handlenullvalues/_main.py

```diff
@@ -115,7 +123,13 @@ def __fill_values_columns(self):
             self.new_df[column].fillna(self.fill_values[column], inplace=True)
         return self.new_df
 
-    def execute(self):
+    # main function
+    def execute(self, params):
+
+        self.__validate_input(params)
+        self.df = params["df"]
+        self.new_df = None
```

Comment:
I think this can be removed as it is not being used anywhere.

Reply:
Yes, will change the code.